src/main/java/com/marginallyclever/makelangelo/plotter/plottercontrols/MarlinPlotterInterface.java

```diff
@@ -203,8 +203,23 @@ private static String getColorName(int toolNumber) {
 			case 0xff00ff -> "magenta";
 			case 0xffff00 -> "yellow";
 			case 0xffffff -> "white";
-			default -> "0x" + Integer.toHexString(toolNumber); // display unknown RGB value as hex
+			default -> "0x" + paddedHex(toolNumber); // display unknown RGB value as hex
 		};
 		return name;
 	}
+
+	/**
+	 * @param number the number to convert to hex
+	 * @return the hex number returned is 6 characters long.  May pad the left end with zeros.
+	 */
+	private static String paddedHex(int number) {
```

Comment:
Can you please move this method to `com.marginallyclever.convenience.StringHelper` and add some unit tests to it ?